test/testdata/compiler/disabled/validate_exp_tests/test.llo.exp

```diff
@@ -0,0 +1,234 @@
+@"str_<top (required)>" = private unnamed_addr constant [17 x i8] c"<top (required)>\00", align 1
+@"rubyStrFrozen_<top (required)>" = internal unnamed_addr global i64 0, align 8
+@"rubyStrFrozen_test/testdata/compiler/disabled/validate_exp_tests/test.rb" = internal unnamed_addr global i64 0, align 8
+@"str_test/testdata/compiler/disabled/validate_exp_tests/test.rb" = private unnamed_addr constant [59 x i8] c"test/testdata/compiler/disabled/validate_exp_tests/test.rb\00", align 1
+@iseqEncodedArray = internal global [6 x i64] zeroinitializer
+@fileLineNumberInfo = internal global %struct.SorbetLineNumberInfo zeroinitializer
+@"rubyStrFrozen_Hello!" = internal unnamed_addr global i64 0, align 8
+@"str_Hello!" = private unnamed_addr constant [7 x i8] c"Hello!\00", align 1
+@ic_puts = internal global %struct.FunctionInlineCache zeroinitializer
+@rubyIdPrecomputed_puts = internal unnamed_addr global i64 0, align 8
+@str_puts = private unnamed_addr constant [5 x i8] c"puts\00", align 1
+
+declare %struct.rb_iseq_struct* @sorbet_allocateRubyStackFrame(i64, i64, i64, i64, %struct.rb_iseq_struct*, i32, i32, %struct.SorbetLineNumberInfo*, i64*, i32, i32) local_unnamed_addr #0
+
+declare void @sorbet_initLineNumberInfo(%struct.SorbetLineNumberInfo*, i64*, i32) local_unnamed_addr #0
+
+declare i64 @sorbet_readRealpath() local_unnamed_addr #0
+
+declare void @sorbet_setupFunctionInlineCache(%struct.FunctionInlineCache*, i64, i32, i32, i32, i64*) local_unnamed_addr #0
+
+declare i64 @sorbet_callFuncWithCache(%struct.FunctionInlineCache*, i64) local_unnamed_addr #0
+
+declare void @sorbet_setMethodStackFrame(%struct.rb_execution_context_struct*, %struct.rb_control_frame_struct*, %struct.rb_iseq_struct*) local_unnamed_addr #0
+
+declare i64 @rb_intern2(i8*, i64) local_unnamed_addr #0
+
+declare i64 @rb_fstring_new(i8*, i64) local_unnamed_addr #0
+
+declare void @rb_gc_register_mark_object(i64) local_unnamed_addr #0
+
+; Function Attrs: noreturn
+declare void @rb_raise(i64, i8*, ...) local_unnamed_addr #1
+
+; Function Attrs: nounwind ssp uwtable
+define weak i32 @sorbet_getIsReleaseBuild() local_unnamed_addr #2 {
+  %1 = load i64, i64* @rb_eRuntimeError, align 8, !tbaa !4
+  tail call void (i64, i8*, ...) @rb_raise(i64 %1, i8* noundef getelementptr inbounds ([93 x i8], [93 x i8]* @.str.10, i64 0, i64 0)) #4
+  unreachable
+}
+
+; Function Attrs: nounwind ssp uwtable
+define weak i8* @sorbet_getBuildSCMRevision() local_unnamed_addr #2 {
+  %1 = load i64, i64* @rb_eRuntimeError, align 8, !tbaa !4
+  tail call void (i64, i8*, ...) @rb_raise(i64 %1, i8* noundef getelementptr inbounds ([95 x i8], [95 x i8]* @.str.9, i64 0, i64 0)) #4
+  unreachable
+}
+
+; Function Attrs: sspreq
+define void @Init_test() local_unnamed_addr #3 {
+entry:
+  %locals.i.i = alloca i64, i32 0, align 8
+
+  ; Call a function that doesn't exist to force a error that won't show up as a diff
+  %realpath = tail call i64 @sorbet_readRealpath_fake()
```

Comment:
So it looks like we have manually injected an error here. Will `update_compiler_exp.sh` clobber this if run?

Reply:
That's a really good point, I wonder if the best way to do this is to just exempt the `disabled/validate_exp_tests` path from exp updates?

Reply:
I think that probably makes sense! Racking my brain a bit, I can't imagine the use of updating `exp` files for a disabled test.

Reply:
I took that approach for now. If there's a better way to maintain the hand-edits, we can make that change in the future :)